staff/ana-sanjuan/mailchimp/styles.css

```diff
@@ -0,0 +1,249 @@
+  outline: 0;
+}
+
+body {
+  font-family: 'PT Sans';
+  background-color: black;
+}
+
+.container-all {
+  margin: 0 11%;
+}
+
+header {
+  background-color: #45BDD4;
+}
+
+.upperheader {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  margin: 1% 5% 1% 5%;
+}
+
+.upperheader__logo {
+  -webkit-box-flex: 30%;
+      -ms-flex: 30%;
+          flex: 30%;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  margin: 4% 0% 0% 2%;
+}
+
+.upperheader__button {
+  background-color: #3F3F37;
+  color: #DDDDDF;
+  width: 120px;
+  font-size: 12px;
+}
+
+.upperheader__nav-container {
+  -webkit-box-flex: 55%;
+      -ms-flex: 55%;
+          flex: 55%;
+  margin: 0% 2% 0% 0%;
+}
+
+.upperheader__p {
+  display: block;
+  text-align: right;
+  margin: 2% 7% 0% 2%;
+  color: white;
+}
+
+.upperheader__p a {
+  text-decoration: none;
+}
+
+.upperheader__nav__ulist {
+  background-color: #CFEEF4;
+  list-style-type: none;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  border-right: #C3DEE3 1px solid;
+  margin-top: 1.1%;
+  border-radius: 10px;
+  border: none;
+  height: 50px;
+  text-shadow: 2px 2px 5px white;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+}
+
+.upperheader__nav__ulist li {
+  border-right: 1px #adc8ca solid;
+  height: 50px;
+  line-height: 50px;
+  text-align: center;
+  padding-left: 5px;
+  padding-right: 20px;
+  text-align: left;
+  font-size: 15px;
+}
+
+.upperheader__nav__ulist a {
+  text-decoration: none;
+}
+
+.upperheader__nav__ulist li:last-child {
+  border: none;
+}
+
+.upperheader__input {
+  background: white url(mailchimp/images/search.png) right no-repeat;
+}
+
+.lowerheader {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  margin: 0 5%;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+}
+
+.lowerheader__image {
+  -webkit-box-flex: 50%;
+      -ms-flex: 50%;
+          flex: 50%;
+  margin: 0 0 4% 5%;
+}
+
+.lowerheader__text {
+  -webkit-box-flex: 40%;
+      -ms-flex: 40%;
+          flex: 40%;
+  color: #11758C;
+  margin-right: 5%;
+  margin-top: 3%;
+}
+
+.lowerheader__button {
+  background-color: #EA5C41;
+  color: #F7F0E4;
+  width: 200px;
+  font-size: 16px;
+}
+
+main {
+  background-image: url("mailchimp/images/noisepattern.png");
+}
+
+.container-main {
+  margin: 0% 8% 0% 5%;
+  background-color: #F0F0F0;
+  position: relative;
+  bottom: 50px;
+  height: 150%;
+  width: 90%;
+  border-radius: 1%;
+}
+
+.container-main h2 {
+  padding: 3% 0 0 1%;
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.main-section {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  margin: 2% 3%;
+  height: 500px;
+}
+
+.main-section h5 {
+  margin: 15% 0 0 0;
```

Comment:
I guess you could use margin-top, and avoid 0 0 0

Reply:
yes, thanks!

Reply:
I need it because I want to specify that the other margins are 0px because they are more by default in h5 headers